docs/source/programming_resources/shared/program_and_manage_network/Connecting-a-Laptop-to-the-Program-&-Manage-Network.rst

```diff
@@ -0,0 +1,131 @@
+Connecting a Laptop to the Program & Manage Network
+===================================================
+
+Connecting Your Laptop to the Program & Manage Network
```

Comment:
Should make header different from title of document

Reply:
It _is_ different.